### PR TITLE
Add more informations about Firebase feature switches

### PR DESCRIPTION
**Pull request by MichalTKwiecien**
We had a couple of issues with people overrided other changes because of force pushing changes, also we've agreed to turn on the switch before the cut-off but it wasn't written anywhere. I've added this and a couple of other things to our Firebase feature switch section.

### CODE REVIEW

File: Cookbook/Technical-Documents/FeatureSwitches.md [hunk 1]
```diff
@@ -94,14 +94,15 @@ Depending on the use case we are using different ways to define Feature Switches
...
-	
+	To control the value of this feature flag we can define values for different "conditions" which are based on the application bundle id and the build version (not the app semantic version number). You can reuse existing conditions (don't mix them with those used for Android app unless you agree to use the same flag for both platforms) or create a new condtion on the `Condtions` page. To create a new condition you need to specify the app bundle identifier and optionally a regular expression for build number (you can use [this tool](http://gamon.webfactional.com/regexnumericrangegenerator/) to create it and [this tool](https://regexr.com) to see if your regular expression works). **Remember to start with local feature switch first when working on a new feature**, read next sections of this article for more details.
```
Comment (DeFrenZ) [inline Cookbook/Technical-Documents/FeatureSwitches.md]: Since we're editing it, might as well fix the typo in `Conditions`

**Review by DeFrenZ — APPROVED** [review]
Having more info is definitely appreciated 👍

File: Cookbook/Technical-Documents/FeatureSwitches.md (hunk 1 above)
Comment (MichalTKwiecien) [inline Cookbook/Technical-Documents/FeatureSwitches.md]:
```suggestion
	To control the value of this feature flag we can define values for different "conditions" which are based on the application bundle id and the build version (not the app semantic version number). You can reuse existing conditions (don't mix them with those used for Android app unless you agree to use the same flag for both platforms) or create a new condition on the `Conditions` page. To create a new condition you need to specify the app bundle identifier and optionally a regular expression for build number (you can use [this tool](http://gamon.webfactional.com/regexnumericrangegenerator/) to create it and [this tool](https://regexr.com) to see if your regular expression works). **Remember to start with local feature switch first when working on a new feature**, read next sections of this article for more details.
```

File: Cookbook/Technical-Documents/FeatureSwitches.md [hunk 2]
```diff
@@ -94,14 +94,15 @@ Depending on the use case we are using different ways to define Feature Switches
...
 	
-
+	If you want to release a feature, set proper conditions on Firebase **before the release cut off day**. Remember to adjust automation tests to make them work regardless if it's turned on or off. When publishing changes in Firebase Console you might be prompted if you want to force save your changes (that could happen when two people made some changes at the same time). **Never force save your changes** - if you see the prompt, cancel your changes, refresh the console and apply them again.
+	
+	After releasing the feature hidden behind the remote feature switch, make some agreement with your PM when we can stop using this switch. Depends how your squad works, you may want to create a ticket in the backlog to phase out the feature switch and remove legacy code. Remember that even you've phased out the feature switch from the newest version, there are still older versions that will be using this feature switch for a long time. After removing it from code, go to the Firebase Console and update description to say which app versions this switch is affecting.
```
Comment (Daniel1of1) [inline Cookbook/Technical-Documents/FeatureSwitches.md]: [Nit | Subjective style opinion] i.e feel free to ignore.
```suggestion
	 After releasing the feature hidden behind the remote feature switch, make some agreement with your PM when we can stop using this switch. Depending on how your squad works, you may want to create a ticket in the backlog to phase out the feature switch and remove legacy code. Remember that even you've phased out the feature switch from the newest version, there are still older versions that could be using this feature switch for a long time. After removing it from code, go to the Firebase Console and update description to say which app versions this switch is affecting.
```

[typo]
```suggestion
	After releasing the feature hidden behind the remote feature switch, make some agreement with your PM when we can stop using this switch. Depends how your squad works, you may want to create a ticket in the backlog to phase out the feature switch and remove legacy code. Remember that even if you've phased out the feature switch from the newest version, there are still older versions that will be using this feature switch for a long time. After removing it from code, go to the Firebase Console and update description to say which app versions this switch is affecting.
```

[Both suggestions combined]
```suggestion
	 After releasing the feature hidden behind the remote feature switch, make some agreement with your PM when we can stop using this switch. Depending on how your squad works, you may want to create a ticket in the backlog to phase out the feature switch and remove legacy code. Remember that even you've phased out the feature switch from the newest version, there are still older versions that could be using this feature switch for a long time. After removing it from code, go to the Firebase Console and update description to say which app versions this switch is affecting.
```

File: Cookbook/Technical-Documents/FeatureSwitches.md [hunk 3]
```diff
@@ -170,3 +171,4 @@ This is yet to be defined.
 
 - Don't change the default value of the flag. We use `false` as default value for all the flags to make their behaviour more predictable and uniform (in opposite to having some flags have it `true` and some have it `false`).
 - Don't introduce feature flags that affect each other. Each feature flag increases testing complexity as it introduces new combinations.
+- Don't name you feature switch `New XXX`. After couple of month you might end up with `New New XXX`. Try to use specific names when possible.
```
Comment (Daniel1of1) [inline Cookbook/Technical-Documents/FeatureSwitches.md]: 👍 `final_v2_updated_final_edited_4_final.psd` 😆

**Review by Daniel1of1 — APPROVED** [review]
Some suggestions, one typo / grammar which I think should be in, one subjective suggestion 😄

**Pull request merged**